packages/web/src/components/ui/PackagePreview/PackagePreview.tsx

```diff
@@ -139,38 +206,66 @@ export default function PackagePreview({ name, version, opened }: Props) {
                   <Icon kind='license' color='#8E8AA0' className={styles.statIcon} />
                   License
                 </div>
-                <div className={styles.statTitle}>MIT license</div>
-                <div className={styles.statSubtitle}>freely distributable</div>
+                {packageDetailsLoading ? (
+                  <>
+                    <Skeleton>
+                      <div className={styles.statTitle}>MIT license</div>
+                    </Skeleton>
+                    <Skeleton>
+                      <div className={styles.statSubtitle}>freely distributable</div>
+                    </Skeleton>
+                  </>
+                ) : (
+                  <>
+                    <div className={styles.statTitle}>MIT license</div>
+                    <div className={styles.statSubtitle}>freely distributable</div>
+                  </>
+                )}
               </div>
 
               <div className={clsx(styles.stat, styles.statListItemSmall)}>
                 <div className={styles.statHeader}>
                   <Icon kind='rating' color='#8E8AA0' className={styles.statIcon} />
                   Rating
                 </div>
-                <div className={styles.statTitle}>
-                  385
-                  {/* or: <div className={clsx(styles.statRating, styles.statRatingRed)}> */}
-                  <div className={clsx(styles.statRating, styles.statRatingGreen)}>
-                    <Icon
-                      kind='ratingArrow'
-                      width={12}
-                      height={12}
-                      className={styles.statRatingArrow}
-                    />
-                    +4
-                  </div>
-                </div>
-                <div className={styles.statSubtitle}>out of 12 842</div>
+                {packageDetailsLoading ? (
+                  <>
+                    <Skeleton>
+                      <div className={styles.statTitle}>385</div>
+                    </Skeleton>
+                    <Skeleton>
+                      <div className={styles.statSubtitle}>out of 12 842</div>
+                    </Skeleton>
+                  </>
+                ) : (
+                  <>
+                    <div className={styles.statTitle}>
+                      385
+                      {/* or: <div className={clsx(styles.statRating, styles.statRatingRed)}> */}
```

Comment:
What about adding a `rankingDelta` prop and deciding on class name based on number's sign?

Reply:
Added as `TODO` for future `PackagePreview` refactor.